osu.Game.Rulesets.Osu/Objects/Drawables/DrawableSpinner.cs

```diff
@@ -177,7 +177,7 @@ protected override void CheckForResult(bool userTriggered, double timeOffset)
         protected override void Update()
         {
             Disc.Tracking = OsuActionInputManager?.PressedActions.Any(x => x == OsuAction.LeftButton || x == OsuAction.RightButton) ?? false;
-            if (!SpmCounter.IsPresent && Disc.Tracking)
+            if (!SpmCounter.IsPresent && (Disc.Tracking || !Disc.Enabled))
```

Comment:
The `Enabled` thing would be pretty good if not for this. In the context of this PR this change seems fine, but when reading the source at face value it makes no sense why the SPM counter should be visible when disabled. This means that the spinner *still* essentially is aware of the mod, but in a round-about way (through a nebulous property).